openwisp_users/tests/utils.py

```diff
@@ -57,7 +50,12 @@ def _create_operator(self, organizations=[], **kwargs):
         )
         opts.update(kwargs)
         operator = User.objects.create_user(**opts)
-        operator.user_permissions.add(*self.get_operator_permissions())
+        groups = Group.objects.filter(name__in=['Administrator', 'Operator'])
```

Comment:
I think I've not understood the issue or solution properly.
If we are creation an operator why `name__in=['Administrator', 'Operator']`, doesn't it just need to be in the Operator group?

Reply:
The [issue](https://github.com/openwisp/openwisp-users/issues/106) specified that I should assign both Administrator and Operator groups.